content/en/references/network-troubleshooting/endpoint-url/_index.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Accessing LocalStack via the Endpoint URL
+weight: 1
+tags:
+- troubleshooting
+- networking
+---
+
+Use this page to find the scenario that best matches your use case
+
+# From the same computer
+
+{{< figure src="../images/1.svg" width="400" >}}
+
+**Example**: you have run `localstack start`, or you are accessing LocalStack started in Docker (or docker-compose).
+
+The hostname `localhost.localstack.cloud` maps to 127.0.0.1, i.e. your computer.
+If you expose port 4566 from your LocalStack container then you should be able to connect.
+If not, you can use `localhost` or any domain name that refers to `localhost`.
+
+# From a container LocalStack created
+
+{{< figure src="../images/4.svg" width="400" >}}
+
+**Example**: your code is running in an ECS container that LocalStack has created.
+
+It may be useful to start LocalStack in a [user-defined network](https://docs.docker.com/network/bridge/), and set the `LAMBDA_DOCKER_NETWORK` environment variable to this value.
```

Comment:
So I'm wondering about the usage of `LAMBDA_DOCKER_NETWORK`. The naming here is really confusing because most of the page doesn't actually even talk about lambda specifically and even references ECS. Maybe we should add a `CONTAINER_DOCKER_NETWORK` (or something else that's more descriptive) for now, to at least make the current behavior clearer?

Reply:
That's certainly a good point about mentioning lambdas, but I think rather than adding another variable, I'd like to solve it in a more seamless way. Thanks for raising, and I'll add some clarity to the document about this point.

Reply:
There is already "MAIN_DOCKER_NETWORK" which is basically the general version of `LAMBDA_DOCKER_NETWORK`.

Reply:
Unfortunately ECS does not use `MAIN_DOCKER_NETWORK` :(

Reply:
https://github.com/localstack/localstack-ext/pull/1426